corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -498,28 +498,22 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         },
 
         addAddressPin: function (geocoder, addressMap, headers, model, addressIndex) {
-            const address = model.attributes.data[addressIndex];
-            return new Promise((resolve) => {
-                geocoder.query(address, function (err, data) {
-                    if (err === null) {
-                        let popupText = "";
-                        headers.forEach((header, index) => {
-                            if (header) {
-                                const valueSanitized = DOMPurify.sanitize(model.attributes.data[index]);
-                                const headerSanitized = DOMPurify.sanitize(header);
-                                popupText += "<b>" + headerSanitized + ":</b> " + valueSanitized + "<br>";
-                            }
-                        });
-                        L.marker(data.latlng)
-                            .addTo(addressMap)
-                            .bindPopup(popupText);
-
-                        resolve(data.latlng);
-                    } else {
-                        resolve(undefined);
-                    }
-                });
+            const coordinates = model.attributes.data[addressIndex];
+            let latLng = coordinates.split(" ").slice(0,2);
```

Comment:
What is `coordinates` for cases with a blank property? If it's null this line will error, and if it's an empty string, looks like `latLng` will end up as `['']` which is truthy and won't get filtered out.

Reply:
Thanks for the catch. Fixed